modules/ROOT/pages/database-administration/standard-databases/configuration-parameters.adoc

```diff
@@ -1,89 +1,6 @@
-* Dropped databases are completely removed and are not intended to be used again at all.
-
-The following Cypher commands are used on the `system` database to manage multiple databases:
-
-[options="header" cols="m,a"]
-|===
-| Command
-| Description
-
-| CREATE DATABASE name
-| Create and start a new database. label:enterprise-edition[]
-
-| DROP DATABASE name
-| Drop (remove) an existing database. label:enterprise-edition[]
-
-| ALTER DATABASE name
-| Alter (modify) an existing database. label:enterprise-edition[]
-
-| START DATABASE name
-| Start a database that has been stopped.
-
-| STOP DATABASE name
-| Shut down a database.
-
-| SHOW DATABASE name
-| Show the status of a specific database.
-
-| SHOW DATABASES
-| Show the name and status of all the databases.
-
-| SHOW DEFAULT DATABASE
-| Show the name and status of the default database.
-
-| SHOW HOME DATABASE
-| Show the name and status of the home database for the current user.
-|===
-
-
-Naming rules for databases are as follows:
-
-* Length must be between 3 and 63 characters.
-* The first character of a name must be an ASCII alphabetic character.
-* Subsequent characters must be ASCII alphabetic or numeric characters, dots or dashes; `[a..z][0..9].-`
-* Names are case-insensitive and normalized to lowercase.
-* Names that begin with an underscore and with the prefix `system` are reserved for internal use.
-
-[NOTE]
-====
-All of the above commands are executed as Cypher commands, and the database name is subject to the link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/syntax/naming[standard Cypher restrictions on valid identifiers].
-In particular, the `-` (dash) and `.` (dot) characters are not legal in Cypher variables, and therefore names with dashes must be enclosed within back-ticks.
-For example, `CREATE DATABASE ++`main-db`++`.
-Database names are the only identifier for which dots don't need to be escaped.
-For example, `main.db` is a valid database name.
-====
-
-It is possible to create an alias to refer to an existing database to avoid these restrictions.
-For more information, see link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/aliases#alias-management-create-database-alias[Cypher Manual -> Creating database aliases].
-
-For detailed information on Cypher administrative commands, see link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control[Cypher Manual -> Access Control].
-
-For examples of using the Cypher administrative commands to manage multiple active databases, see xref:manage-databases/queries.adoc[Queries].
-
-
+:description: This page describes the relevant configuration parameters for managing databases with Neo4j.
 [[manage-databases-parameters]]
-== Configuration parameters
+= Configuration parameters
 
 Configuration parameters are defined in the xref:configuration/neo4j-conf.adoc[neo4j.conf] file.
```

Comment:
there is at least one link down below (for alter database in the `server.databases.read_only` section) that needs updating, might be more as well but I didn't check)